src/main/kotlin/com/openlattice/assembler/Assembler.kt

```diff
@@ -274,14 +276,51 @@ class Assembler(
 
     @Subscribe
     fun handleAddMembersToOrganization(event: MembersAddedToOrganizationEvent) {
+        // check if organization is initialized
+        ensureAssemblyInitialized(event.organizationId)
```

Comment:
I added an EntryProcessor you can use in this method to avoid serializing the whole `OrganizationAssembly` just for a boolean check: https://github.com/openlattice/conductor-client/pull/619/files#diff-d008436a30debc36dd15f70fe40cfb74

Reply:
Nice, thanks! Will do after merge!